applications/db_development_tools/add_unit_to_parameter_in_db.py

```diff
@@ -24,40 +24,40 @@ def main():
     parser.initialize_default_arguments(add_workflow_config=False)
 
     args = parser.parse_args()
-    cfg.setConfigFileName(args.configFile)
+    cfg.set_config_file_name(args.config_file)
 
     logger = logging.getLogger()
-    logger.setLevel(gen.getLogLevelFromUser(args.logLevel))
+    logger.setLevel(gen.get_log_level_from_user(args.log_level))
```

Comment:
setLevel didn't go to snake case here

Reply:
Wait, before you continue pointing out all of them, that's a logging function, not ours, so I obviously can't change it.